examples/contrib/sops/README.md

```diff
@@ -24,23 +26,47 @@ or if your keys are already in `gpg`, it's possibe to run:
 kpt fn run --env SOPS_IMPORT_PGP="$(gpg --armor --export-secret-keys)" <folder>
 ```
 
+For AGE case this function introduces and additional `SOPS_IMPORT_AGE` ENV var
+that must contain the [SOPS age keys.txt file](https://github.com/mozilla/sops/blob/master/age/keys.txt).
+This file is needed for decryption. For encryption it's possible to use
+`SOPS_AGE_RECIPIENTS` ENV variable or parameter `age`.
+
+```sh
+kpt fn run --env SOPS_IMPORT_AGE="$(cat <file with age keys>.txt)" <folder>
+```
+
+of if SOPS is already working , it's possible to run:
+
+```sh
+kpt fn run --env SOPS_IMPORT_AGE="$(cat ~/.config/sops/age/keys.txt)" <folder>
+```
+
 ## Function invocation
 
 Get this example and try it out by running the following commands:
 
 ```sh
 # download sops kpt-function example
 kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/contrib/sops .
-# copy example key from sops project
-curl -fsSL -o sops/key.asc https://raw.githubusercontent.com/mozilla/sops/master/pgp/sops_functional_tests_key.asc
-# run the function
-kpt fn run --env SOPS_IMPORT_PGP="$(cat sops/key.asc)" sops/local-configs
+cd kpt-functions-catalog/examples/contrib/sops
+
+# copy example GPG key from sops project
+curl -fsSL -o gpg_keys.asc https://raw.githubusercontent.com/mozilla/sops/master/pgp/sops_functional_tests_key.asc
+# copy example AGE key from sops project
+curl -fsSL -o age_keys.txt https://raw.githubusercontent.com/mozilla/sops/master/age/keys.txt
+
+# run the function to work with GPG
+kpt fn run --env SOPS_IMPORT_PGP="$(cat gpg_keys.asc)" gpg
+# run the function to work with AGE
+kpt fn run --env SOPS_IMPORT_AGE="$(cat age_keys.txt)" age
 ```
 
 ## Expected result
 
 Verify the updated configuration:
 
 ```sh
-kpt cfg cat sops/local-configs
+kpt cfg cat gpg
```

Comment:
Our current model for examples is that each example should have one package and one README.md.
I guess it probably makes more sense to split it into 2 examples.

Reply:
@mengqiy , thank you for reviewing that!
I've split them, and now there is a following structure

```
examples/contrib/sops/
├── age
│   ├── function.yaml
│   ├── README.md
│   ├── to-decrypt.yaml
│   └── to-encrypt.yaml
├── gpg
│   ├── function.yaml
│   ├── README.md
│   ├── to-decrypt.yaml
│   └── to-encrypt.yaml
└── README.md
```

But now I'm reading your comment again and thinking about a bit different approach:

```
examples/contrib/
├── sops-age
│   ├── function.yaml
│   ├── README.md
│   ├── to-decrypt.yaml
│   └── to-encrypt.yaml
├── sops-gpg
│   ├── function.yaml
│   ├── README.md
│   ├── to-decrypt.yaml
│   └── to-encrypt.yaml
```

The only concern with that - no place to store some common info about the module. And I think - duplicating it into both README's isn't a good idea as well.

One more option is to combine 2 examples into 1, but this may be too complex.

Please let me know what do you think about the current option. I'm ready to modify it based on you suggestions  :)